src/policy/inputs.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// NOTE: This file is intended to be customised by the end user, and includes only local node policy logic
+
+#include "policy/policy.h"
```

Comment:
I think it would be beneficial to complete the split, i.e.:

- include `script/standard.h` instead
- move `MAX_P2SH_SIGOPS`, `MAX_STANDARD_P2WSH_STACK_ITEMS`, `MAX_STANDARD_P2WSH_STACK_ITEM_SIZE` and `MAX_STANDARD_P2WSH_SCRIPT_SIZE` from `policy.h` to `inputs.h`.